desc/nestor.py

```diff
@@ -0,0 +1,1287 @@
+"""Neumann Solver for Toroidal Systems.
+
+Created on Mon Jun  7 13:15:05 2021
+
+@author: Jonathan Schilling (jonathan.schilling@ipp.mpg.de)
+"""
+
+import numpy as np
+from scipy.constants import mu_0
+
+from desc.backend import fori_loop, jnp, put
+from desc.grid import LinearGrid
+from desc.io import IOAble
+from desc.transform import Transform
+from desc.utils import Index
+
+
+def copy_vector_periods(vec, zetas):
```

Comment:
This is Jonathan's code so maybe we won't change it, but don't we have util functions for this already?

Reply:
Not that I know of? This is basically rotating a vector from a single field period to the other periods.